### PR TITLE
Release version 1.4.13

### PR DESCRIPTION
**Pull request by emelvarol-cognite**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1038,6 +1053,87 @@ spark.sql(
...
 ```
 
+
+### Relationships
+
+Learn more about relationships [here](https://docs.cognite.com/api/v1/#tag/Relationships)
+
+Note that relationships can not be updated, but can only be read, created, or deleted.
+If you want to change a relationship, you can first delete it, and then recreate it
+with the same external id. externalId, sourceExternalId, sourceType, targetExternalId,
```
Comment (hakontro) [inline README.md:1063]: Let's use formatting like `externalId` when refering to field names.

**Review by hakontro — CHANGES_REQUESTED** [review]
Looks good! Can you update the changelog too?

File: README.md [hunk 2]
```diff
@@ -1038,6 +1053,87 @@ spark.sql(
...
+
+### Relationships
+
+Learn more about relationships [here](https://docs.cognite.com/api/v1/#tag/Relationships)
+
+Note that relationships can't be updated, but can only be read, created, or deleted.
+If you want to change a relationship, you can first delete it, and then recreate it
+with the same external id. `externalId`, `sourceExternalId`, `sourceType`, `targetExternalId`, 
+`targetType` can't be null.
+
```
Comment (emelvarol-cognite) [inline README.md:1065]: @hakontro

**Comment by emelvarol-cognite** [comment]
> Looks good! Can you update the changelog too?

I did not see this. Will do now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# 1.4.13
+
+## Enhancements
+* `relationships` have been added as a new resource type. See [relationships](README.md#relationships)
+for more information.
+* `labels` field is now available for assets on read and insert operations.
+
```
Comment (emelvarol-cognite) [inline CHANGELOG.md:7]: @hakontro

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
+# 1.4.13
+
+## Enhancements
+* `relationships` have been added as a new resource type. See [relationships](README.md#relationships)
+for more information.
+* `labels` field is now available for assets on read and insert operations.
```
Comment (hakontro) [inline CHANGELOG.md:6]:
```suggestion
* The `labels` field is now available for assets on read and insert operations.
```

**Review by hakontro — APPROVED** [review]
Great!

**Pull request merged**